javascript/builtins/Intl.json

```
@@ -414,6 +414,56 @@
               "deprecated": false
             }
           },
+          "dateStyle": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "76"
+                },
+                "chrome_android": {
+                  "version_added": "76"
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "nodejs": {
+                  "version_added": null
```

NodeJS is powered by V8, Chrome's own JavaScript engine.  While I'm not really sure what NodeJS version it's in (and we're not really focusing on NodeJS right now), I think setting it to `true` would be better!

```suggestion
                  "version_added": true
```